projects/legacy/components/input-time/input-time.component.ts

```diff
@@ -81,6 +85,12 @@ export class TuiInputTimeComponent
     @Input()
     public mode: TuiInputTimeOptions['mode'] = this.options.mode;
 
+    public readonly identityMatcher?: TuiIdentityMatcher<TuiTime> = this.valueTransformer
```

Comment:
Maybe just have it regardless? So that we have less branching in the code

Reply:
It might affect performance or the existing behavior. Remove the condition anyway?

Reply:
Yes, let's go for it. This difference is insignificant.